x/evm/types/key.go

```diff
@@ -30,9 +30,9 @@ const (
 	// The EVM module should use a prefix store.
 	StoreKey = ModuleName
 
-	// TransientKey is the key to access the EVM transient store, that is reset
+	// ObjectKey is the key to access the EVM transient store, that is reset
```

Comment:
it's object store right, should we change to `ObjectStoreKey` to be consistent?

Reply:
done